docs/configure/toolbar.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Toolbar'
+---
+
+<YouTubeCallout id="TODO?" title="How to Configure the Toolbar" />
+
+Storybook’s toolbar appears at the top of stories and docs pages. It displays menus from addons that can customise the preview environment, like the [viewport] and [background]) addons, as well as [custom menus]() you've defined.
```

Comment:
Please provide URLs for all links. They should be relative and include the `.md` extension (so they function on GitHub):

```suggestion
Storybook’s toolbar appears at the top of stories and docs pages. It displays menus from addons that can customize the preview environment, like the [viewport](../essentials/viewport.md) and [background](../essentials/background.md) addons, as well as [custom menus](../essentials/toolbars-and-globals.md#global-types-and-the-toolbar-annotation) you've defined.
```

Reply:
Thanks for catching this mistake! I'm committing this locally to spare me extra git operations and to avoid extra CI runs.